app-services/secrets/functions/getsecrets.go

```diff
@@ -30,69 +30,43 @@ type getSecretsTestData struct {
 }
 
 var secureSecretsData = []getSecretsTestData{
```

Comment:
Ok, I think i get what you are doing here, but calling it `testData` throws me off. Something more like `getSecretsTestData` => `expectedSecret` and  `secureSecretsData` => `expectedSecrets`. Also remove the look and fell of a unit test. `testName` -> `name` and remove `errExpected`